lib/room.rb

```diff
@@ -0,0 +1,4 @@
+module BookingLogic
+  class Room
+  end
+end
```

Comment:
Don't forget to remove unused files before you submit your project/as you refactor them out!